mycroft/skills/phillips_hue/__init__.py

```diff
@@ -0,0 +1,283 @@
+        self.register_intent(decrease_color_temperature_intent,
+                             self.handle_intent)
+
+        increase_color_temperature_intent =\
+            IntentBuilder("IncreaseColorTemperatureIntent")\
+            .require("IncreaseColorTemperatureKeyword").build()
+        self.register_intent(increase_color_temperature_intent,
+                             self.handle_intent)
+
+        connect_lights_intent = \
+            IntentBuilder("ConnectLightsIntent") \
+            .require("ConnectLightsKeyword").build()
+        self.register_intent(connect_lights_intent,
+                             self.handle_intent)
+
+    def handle_intent(self, message):
+        if message.message_type == 'ConnectLightsIntent'\
+                or self.connected or self._connect_to_bridge():
+            try:
+                if message.message_type == "TurnOffIntent":
+                    self.handle_turn_off_intent(message)
+                elif message.message_type == "TurnOnIntent":
+                    self.handle_turn_on_intent(message)
+                elif message.message_type == "ActivateSceneIntent":
+                    self.handle_activate_scene_intent(message)
+                elif message.message_type == "DecreaseBrightnessIntent":
+                    self.handle_decrease_brightness_intent(message)
+                elif message.message_type == "IncreaseBrightnessIntent":
+                    self.handle_increase_brightness_intent(message)
+                elif message.message_type == "DecreaseColorTemperatureIntent":
+                    self.handle_decrease_color_temperature_intent(message)
+                elif message.message_type == "IncreaseColorTemperatureIntent":
+                    self.handle_increase_color_temperature_intent(message)
+                elif message.message_type == "ConnectLightsIntent":
+                    self.handle_connect_lights_intent(message)
+                else:
+                    raise Exception('No matching intent handler')
+            except Exception as e:
+                if isinstance(e, PhueRequestTimeout):
+                    self.speak_dialog('unable.to.perform.action')
+                elif 'No route to host' in e.args:
+                    if not self.config.get('ip', '') == '':
+                        self.speak_dialog('no.route')
+                        return
+                else:
+                    raise
+
+    def handle_turn_off_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('turn.off')
+        self.all_lights.on = False
+
+    def handle_turn_on_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('turn.on')
+        self.all_lights.on = True
+
+    def handle_activate_scene_intent(self, message):
+        keyword_len = len(message.metadata['ActivateSceneKeyword'])
+        scene_name = message.metadata['utterance'][keyword_len:].strip()
```

Comment:
No problem, it's just something that you have to get used to. I would recommend fairly frequently rebasing off of master so that you have the latest changes and can see if there are any conflicts.